src/api/operations/enrollment.fragments.graphql

```diff
@@ -62,6 +62,9 @@ fragment EnrollmentFields on Enrollment {
     id
     name
   }
+  moveInAddresses {
+    ...ClientAddressFields
+  }
```

Comment:
I think this only needs to go on the `AllEnrollmentDetails` fragment below. That's the one we fetch for the Enrollment dashboard.

Reply:
Good point! I added it to the EnrollmentOccurrencePointFields; so that it gets refreshed after the enrollment details are changed.